README.md

```diff
@@ -1,2 +1,11 @@
 # docker-spark-gpg
-Docker Alpine image with Spark, Python, Java and gpg. Used for local dev environment.
+Docker Alpine image with Spark, Python, Java and gpg. Used for loacal dev environment.
+
+Instructions
+
+7. Run the command in order to mount the export data folder on to the root of the Docker container
```

Comment:
This should mention the 3 directories that need mounting, also renumber these, 7 and 8 no longer make sense